inst/WORDLIST

```diff
@@ -1,84 +1,965 @@
+FIMSMath
+FIMSObject
+FIMSRcppInterfaceBase
+FisheryModelBase
+FisheryModelInterfaceBase
+fishres
+FITFims
+FIXME
+FleetInitializeWorks
+FleetInterface
+FleetInterfaceBase
+FleetPrepareWorks
+FleetTests
+fmort
+Fmort
+FMort
+fn
+frac
+fromJSON
+fromR
+fromRVector
+fsel
+fstream
+func
+FUNC
+FunctionName
 functors
+gcc
+Gcov
+gcovr
 gdb
+getAbsolutePathWithoutDotDot
+GetArray
+GetBool
+GetDouble
+getenv
+GetFixedEffectsParameters
+GetId
+GetInstance
+GetInt
+getline
+GetNages
+GetNseasons
+GetNyears
+GetObject
+GetObservedAgeCompDataID
+GetObservedIndexDataID
+GetObservedLandingsDataID
+GetObservedLengthCompDataID
+getOption
+GetParameters
+GetPopulationIds
+GetPopulations
+getpwuid
+GetRandomEffectsParameters
+GetString
+Getter
+GetType
+getuid
+GetUserNameA
+getwd
 ggplot
+gh
+gha
+ghactions
 github
+gitignore
+glm
+GlobalEnv
+globalVariables
+GMACS
+GNUC
+googlebenchmark
 googletest
+GoogleTest
+grepl
 grey
+growthBase
+GrowthBase
+GrowthEvaluate
+GrowthInterfaceBase
+gsub
 gtest
+HandlesCorrectInput
+HandlesEdgeCase
+HardwareConcurrency
+holt
 hpp
+hpux
+HPUX
+href
+https
+iage
+IBMCPP
+idx
+ifdef
+ifelse
+ifndef
+ifstream
+ilist
+imax
+img
+importFrom
+indentLevel
+IndexDataInterface
+IndexExpected
+IndexNumberAtAge
+IndexNumberAtLength
+infile
+inheritParams
 init
+initializer
+inplace
+InputIt
+inQuotes
+instantiations
+IntegerAgeInput
+IntegerVector
+IntegrationTest
+inv
+InvLogit
+InvLogitLogit
+io
+iostream
+IRIX
+isdigit
+isDouble
+isnan
+isspace
+istringstream
+iter
+itewaa
+JArray
+jmax
+jnll
+johnson
 json
+JSON
+JsonArray
+jsonlite
+JsonObject
+JsonParser
+jsonValue
+JsonValue
+JsonValueType
+jt
+kable
+kelli
+kellijohnson
+kmax
+knitr
+knum
+LandingsDataInterface
+LandingsExpected
+LandingsNumberAtAge
+LandingsNumberAtLength
+LastTest
+lcomp
+len
 lengthcomp
+LengthComp
+LengthCompDataInterface
+LengthCompositionExpected
+LengthCompositionProportion
+lgamma
+libname
+libpath
+LIBPFM
+Lifecycle
+Linf
+linux
+llvm
+lmax
+Lmcons
+loadModule
+loadNamespace
+LogDevs
+LogDevsRecruitmentInterface
+LogDevsRecruitmentProcess
+LogEntry
+logfmort
+LogisticMat
+LogisticMaturity
+LogisticMaturityInterface
+LogisticSel
+LogisticSelectivity
+LogisticSelectivityInterface
+LogitInvLogit
+LogNormal
+LognormalLPDF
+LogNormalLPDF
+logr
+logR
+LogR
+LogRecDev
+LogRRecruitmentInterface
+LogRRecruitmentProcess
 lpdf
+lpmf
+LPMF
+lt
+lte
+lw
+macintosh
+macos
+macOS
+MACOS
+maindir
+MakeADFun
+MakeAvailable
+mapply
+MATHJAX
+mathrm
+maturityAA
+MaturityAtAge
+MaturityBase
+MaturityInterfaceBase
+maxF
+maxgc
+maxgrad
+maxit
+MCP
+MCPC
+md
+meanlog
+mget
 minimizer
+minimizR
+mkdir
+mkString
+ModularTMBExample
+mort
+mpi
+msupernaw
+msy
+mult
+MultinomialLPDF
+MultinomialLPMF
+mutex
+mv
+MVNORM
+mymap
+na
 naa
-nlmimb
+nages
+Nages
+nal
+nan
+NaN
+nathanvaughan
+nCAA
+ncases
+nchar
+ncol
+NCOL
+nd
+NetBSD
+newpage
+nF
+nfleets
+nFmax
+NilValue
+ninput
+nlengths
+nll
+NLL
+nlls
+NLLs
+nlminb
+nlog
+nm
+nMaturity
+nmfs
+NMFS
+nmfspalette
+nMortality
+noaa
```

Comment:
every instance is in a webpage or an email and could be changed to NOAA

Reply:
Hmmm, I’m not sure we want to manually change all `noaa` to `NOAA`. Relying on developers to remember to do that every time they type an email or paste a web link into the codebase in the future seems risky. Maybe I can modify the GHA from {ghactions4r} to normalize the case before running the spelling check against the wordlist. I can post an issue on {ghactions4r} and will revisit the WORDLIST in FIMS after making changes there.